### DISCUSSION TITLE
Equivalent_Strain in Correspondence Models

### DISCUSSION THREAD

**Discussion by ppandit95** [peridigm/peridigm#250] [Q&A]
Dear Peridigm Community

I need to plot a graph of von misses stress vs equivalent_strain at the centre point in a simulation so I was wondering is there a way to output equivalent_strain in yaml file or do I need to use Compute Class for this purpose.Any suggestions in this regard will be helpful.

Many Thanks
Pushkar

**Comment by djlittl** [comment]
As far as I know, you can't request equivalent_strain directly, but you can request Deformation_Gradient.  I think this would allow you to compute equivalent_strain in a post-processing step.  If you want Peridigm to output it directly, you'd need to implement a new compute class.

**Reply by ppandit95** [reply]
Thanks @djlittl for your reponse on above query....

**Comment by johntfoster** [comment]
If the material model uses equivalent plastic strain as a history variable,
it should be able to be queried.  For example, in the "Isotropic Hardening
Coorespondence Material" there is a field variable
"Equivalent_Plastic_Strain" that can be output.

**Reply by ppandit95** [reply]
Thanks @johntfoster for elaborating on that as it will be beneficial to look for characteristics of kinematic hardening such as Bauschinger Effect in Von Mises Stress Vs Equivalent Plastic Strain graph
